_layouts/base.html

```diff
@@ -66,7 +66,11 @@
       apiKey: 'fdd89b9b35b9bd347f34111dbc1c6219',
       indexName: 'dita-ot',
       inputSelector: '#search input[type=text]',
-      debug: false // Set debug to true if you want to inspect the dropdown
+      algoliaOptions: {
+             hitsPerPage: 10,
+             // See https://www.algolia.com/doc/api-reference/api-parameters/
```

Comment:
Since this code runs on every page load, I think we should remove the comments. I'll push a new commit.